docs/advanced-features/workflows.md

```diff
@@ -200,12 +210,32 @@ quilt3.Package().push(
 # Package test/package@06b2815 pushed to s3://quilt-sergey-dev-metadata
```

Comment:
any chance we can make the S3 bucket name more end-user sanitized? Nothing against Sergey by any means, but I'd like this to be something along the lines of s3://quilt-docs-metadata

Reply:
Removed all occurrences of Sergey's name